pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>guava</artifactId>
             <version>18.0</version>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
```

Comment:
Looks like we only need this dependency to have access this:
```
        Response.Status.Family responseFamily = Response.Status.Family.familyOf(actionResponse.getStatus());
```
I would remove the dependency and perform the check manually.